utils.py

```diff
@@ -359,6 +376,19 @@ def prepare_vesting(chainspec):
     rtm['vesting']['vesting'] = vesting
 
 
+def prepare_benchmark_accounts(chainspec, n_of_accounts, azero_amount):
+    bench_accounts = generate_accounts_from_seeds((str(i) for i in range(n_of_accounts)))
+    balances = []
+    azero = 1000000000000
```

Comment:
replace with `1e12`, is less error prone.

Reply:
extracted `azero` to function that returns 1e12 (it was used in couple of places)